src/features/presence/PresenceAvatarTooltipContent.tsx

```diff
@@ -14,11 +14,12 @@ export type Props = {
 } & WrappedComponentProps;
 
 function PresenceAvatarTooltipContent({ name, interactedAt, interactionType, intl, isActive }: Props): JSX.Element {
-    const lastActionMessage = determineInteractionMessage(interactionType);
+    const lastActionMessage = determineInteractionMessage(interactionType, interactedAt);
+    const { unit, value } = convertMillisecondsToUnitAndValue(Math.abs(interactedAt - Date.now()));
```

Comment:
maybe swap the order?
```suggestion
    const { unit, value } = convertMillisecondsToUnitAndValue(Date.now() - interactedAt);
```

then you don't need the Math.abs since Date.now() should always be >= than interactedAt